The execution output will highlight fake errors and warnings

Demo:

```rust
fn main() {
    let _: std::error::Error = "warning:🤔...";
}
```

![screenshot_2018-06-18 17 28 45_gscxcz-fs8](https://user-images.githubusercontent.com/103023/41529186-d3e70ea6-731e-11e8-8b5c-20e41189a545.png)

This also affects runtime output to stderr e.g.

```rust
fn main() {
    eprintln!("warning:🤔...\nerror[E9999]:👀...");
}
```

--- a/ui/frontend/highlighting.ts
+++ b/ui/frontend/highlighting.ts
@@ -2,9 +2,9 @@ import Prism from 'prismjs';
 
 export function configureRustErrors({ gotoPosition, getChannel }) {
   Prism.languages.rust_errors = { // eslint-disable-line camelcase
-    'warning': /warning:.*\n/,
+    'warning': /^warning:.*$/m,
     'error': {
-      pattern: /error(\[E\d+\])?:.*\n/,
+      pattern: /^error(\[E\d+\])?:.*$/m,
       inside: {
         'error-explanation': /\[E\d+\]/,
         'see-issue': /see issue #\d+/,

